app/views/events/_upcoming_events.html.erb

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" />
+    <style>
+      .panel-image img{
+        width: 100%;
+      }
+    </style>
+
+</head>
```

Comment:
If this file is supposed to be here, you should get rid of the head because that will come from the application erb (i assume you were using this for testing purposes because you hadn't rebased master with the pages etc.?